tests/test_agent_manager.py

```diff
@@ -446,46 +446,19 @@ async def test_api(init_dataclasses_and_load_schema):
     assert result == 404
 
     code, all_agents = await am.list_agents(None)
-    assert code == 200
-    shouldbe = {
-        "agents": [
-            {"name": "agent1", "paused": True, "last_failover": "", "primary": "", "environment": env.id, "state": "paused"},
-            {
-                "name": "agent1",
-                "paused": False,
-                "last_failover": expiration,
-                "primary": "",
-                "environment": env5.id,
-                "state": "down",
-            },
-            {"name": "agent2", "paused": False, "last_failover": UNKWN, "primary": UNKWN, "environment": env.id, "state": "up"},
-            {"name": "agent3", "paused": False, "last_failover": UNKWN, "primary": UNKWN, "environment": env.id, "state": "up"},
-            {"name": "agent1", "paused": True, "last_failover": "", "primary": "", "environment": env4.id, "state": "paused"},
-            {
-                "name": "agent2",
-                "paused": False,
-                "last_failover": UNKWN,
-                "primary": UNKWN,
-                "environment": env4.id,
-                "state": "up",
-            },
-            {"name": "agent3", "paused": False, "last_failover": "", "primary": "", "environment": env4.id, "state": "down"},
-            {"name": "agent4", "paused": False, "last_failover": "", "primary": "", "environment": env2.id, "state": "down"},
-        ]
-    }
-    assert_equal_ish(shouldbe, all_agents, sortby=["environment", "name"])
+    assert code == 404
```

Comment:
This is a behavior change: I don't allow the environment to be None, but I suspect the API library would not allow it either